lib/bootstrap_form/form_builder.rb

```diff
@@ -374,14 +374,14 @@ def generate_label(id, name, options, custom_label_col, group_layout)
 
       options[:class] = classes.compact.join(" ")
 
+      label_text = options[:text]
       if label_errors && has_error?(name)
         error_messages = get_error_messages(name)
         label_text = (options[:text] || object.class.human_attribute_name(name)).to_s.concat(" #{error_messages}")
         label(name, label_text, options.except(:text))
-      else
-        label(name, options[:text], options.except(:text))
       end
-
+      
+      label(name, label_text.try(:html_safe), options.except(:text))
```

Comment:
Stretching it here, but if a programmer had an application where some labels were actually built from user-entered data, then a malicious user could craft a label to mount an attack. Would it be sufficient to simply say that labels that contain html have to be html safe before they're passed to the helper?